firestore/client/firestore_client.bzl

```diff
@@ -1,9 +0,0 @@
-# DO NOT EDIT -- GENERATED BY CMake -- Change the CMakeLists.txt file if needed
```

Comment:
ooops.  I asked you to remove this file before #519 got merged, now it is, please do not remove it.

Reply:
Shall I add these files in my PR.

Reply:
Your goal should be to leave these files in the same state they are in the branch you are merging to.

If your branch was rebased to c3cbd8f67d92af3d351cf9989ac4b27e5aacf541 (or any newer commit on `master`), then the files are in `master` and you should not erase them, and you should have no need to modify them in any way.

If your branch was based before that commit then you should not add them.